FantasmoSDK/Classes/FMLocationManager.swift

```diff
@@ -151,16 +153,19 @@ open class FMLocationManager: NSObject, FMApiDelegate {
     
     /// Starts the generation of updates that report the user’s current location.
     public func startUpdatingLocation() {
+        precondition(isConnectedToServer, "Connecting to location service was not performed!")
+        
         log.debug()
-        isConnected = true
-        state = .localizing
+        self.isLocationUpdateInProgress = true
```

Comment:
`self.` is understood. Why add it?

Reply:
Yes, I agree. XCode highlights properties. I have corrected.